samri/optimization/registration.py

```diff
@@ -108,14 +109,21 @@ def structural_rigid_affine(template="/Users/marksm/GitHub/mriPipeline/ants_test
 	n4.inputs.dimension = 3
 	n4.inputs.input_image = input_image
 	# correction bias is introduced (along the z-axis) if the following value is set to under 85. This is likely contingent on resolution.
-	n4.inputs.bspline_fitting_distance = 100
-	n4.inputs.shrink_factor = 2
+	n4.inputs.bspline_fitting_distance = 10
+	# n4.inputs.shrink_factor = 2
 	n4.inputs.n_iterations = [200,200,200,200]
 	n4.inputs.convergence_threshold = 1e-11
 	# n4.inputs.output_image = 'ss_n4_{}_ofM{}.nii.gz'.format(participant,i)
-	n4.inputs.output_image = 'hard.nii.gz'
+	n4.inputs.output_image = 'hard_flirt.nii.gz'
 	n4_res = n4.run()
 
+	flt = fsl.FLIRT(cost_func='corratio', dof = 6, searchr_x = [-180,180], searchr_y = [-180,180], searchr_z = [-180,180], force_scaling = True)
```

Comment:
are `searchr_x` and friends the parameters that make the registration try to reorient the image?